### PR TITLE
Fix linting on delphi utils

### PR DESCRIPTION
**Pull request by chinandrew**
Summary of changes:
- Fix linting across utils. Math-y variable names were ignored.

One outstanding linting error not included in this PR, will be fixed by #460

### CODE REVIEW

File: _delphi_utils_python/delphi_utils/geomap.py [hunk 1]
```diff
@@ -796,7 +796,7 @@ def convert_zip_to_hrr(self, data, zip_col="zip", hrr_col="hrr"):
         return data
 
     def convert_zip_to_msa(
-        self, data, zip_col="zip", msa_col="msa", date_col="date", count_cols=None
+        self, data, zip_col="zip", date_col="date", count_cols=None
```
Comment (krivard) [inline _delphi_utils_python/delphi_utils/geomap.py:799]: What's going on here?

Comment (chinandrew) [reply _delphi_utils_python/delphi_utils/geomap.py:799]: that variable is unused in that function.

Comment (krivard) [reply _delphi_utils_python/delphi_utils/geomap.py:799]: Do we know if any indicators call it with that keyword?

Comment (chinandrew) [reply _delphi_utils_python/delphi_utils/geomap.py:799]: Didn't find any uses when I searched, only saw use of the `zip_to_msa` function below it, which calls `convert_zip_to_msa`.

**Review by krivard — APPROVED** [review]
lgtm

**Pull request merged**